lib/user_search_string.rb

```diff
@@ -0,0 +1,6 @@
+class UserSearchString < String
```

Comment:
I prefer the class method approach on this as well, `UserSearchString.from(search_names)`.